engine/storage/snapshot/src/main/java/org/apache/cloudstack/storage/snapshot/StorageSystemSnapshotStrategy.java

```diff
@@ -354,10 +362,9 @@ public boolean revertSnapshot(SnapshotInfo snapshotInfo) {
 
                 s_logger.error(errMsg);
 
-                throw new CloudRuntimeException(errMsg);
+                throw new CloudRuntimeException("Failed to revert a volume to a snapshot state");
```

Comment:
This seems like a very generic message. Can't we add something like the volume name?

Reply:
Good point, I will update this soon. Thanks!